cleanlab/datalab/datalab.py

```diff
@@ -308,11 +329,18 @@ def report(
         """
         if verbosity is None:
             verbosity = self.verbosity
-        reporter = Reporter(
+        if self.data_issues.issue_summary.empty:
+            print(
+                "Please specify some issue types to run in datalab.find_issues() to see the report.\n"
+            )
+            return
+
+        reporter = report_factory(self._imagelab)(
             data_issues=self.data_issues,
             verbosity=verbosity,
             include_description=include_description,
             show_summary_score=show_summary_score,
+            imagelab=self._imagelab,
```

Comment:
If it's not easy, then you can ignore the rest of my comments.

Reply:
This is not possible, because factory would only return the reporter class, it would still need imagelab object to print its report. But I have removed it as from the `Reporter` class